tests/Feature/MissingDatabaseBackupNotificationTest.php

```
@@ -0,0 +1,124 @@
+it('notifies once per period without executions and rearms after another execution', function () {
+    Notification::fake();
+    $team = teamWithBackupFailureNotifications();
+    $backup = missingBackupSchedule($team);
+
+    (new CheckMissingDatabaseBackupsJob)->handle();
+    (new CheckMissingDatabaseBackupsJob)->handle();
+
+    Notification::assertSentToTimes($team, BackupMissing::class, 1);
+
+    Carbon::setTestNow(now()->addMinute());
+    $execution = ScheduledDatabaseBackupExecution::create([
```

PHPStan (2.2.7)

[warning] Call to an undefined static method App\Models\ScheduledDatabaseBackupExecution::create().


(staticMethod.notFound)